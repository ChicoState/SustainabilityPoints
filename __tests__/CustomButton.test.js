import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import CustomButton from '../components/CustomButton.js';
import renderer from 'react-test-renderer';



describe(CustomButton, () => {
 it('CustomButton renders title', () => {
   const customButton= shallow(<CustomButton title={"Save"} />);
 });
});


test('renders correctly', () => {
  const tree = renderer.create(<CustomButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
