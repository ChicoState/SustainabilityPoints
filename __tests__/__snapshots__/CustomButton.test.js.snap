// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2AC062",
      "borderRadius": 5,
      "display": "flex",
      "height": 50,
      "justifyContent": "center",
      "opacity": 1,
      "shadowColor": "#2AC062",
      "shadowOffset": Object {
        "height": 10,
        "width": 0,
      },
      "shadowOpacity": 0.4,
      "shadowRadius": 20,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#FFFFFF",
          "fontSize": 16,
          "textTransform": "uppercase",
        },
        Object {},
      ]
    }
  />
</View>
`;
